src/components/AiMarketplace/ServiceListingHeader/index.js

```diff
@@ -21,66 +21,70 @@ class ServiceListingHeader extends Component {
   render() {
     const { classes, carousel } = this.props;
 
+    const rightAlign = "RIGHT";
```

Comment:
create it as an enum.. 
like 
IMAGE_ALIGNMENT = {
RIGHT:"RIGHT",
LEFT:"LEFT"}